src/iop/colorin.c

```diff
@@ -56,10 +56,6 @@
 
 #include <lcms2.h>
 
-// max iccprofile file name length
-// must be in synch with dt_colorspaces_color_profile_t
-#define DT_IOP_COLOR_ICC_LEN 512
```

Comment:
No please, not in any of the iop because they must be self sufficient and the value should not be changed without updating the legacy_params() routine. So changing it globally would break IOP having it recorded into their params.

Reply:
OK... then it seems like "dedup" isn't really an option, just a bit of cleaning needed.